test/library/standard/Random/bradc/randomAcrossTypes.chpl

```diff
@@ -52,11 +52,11 @@ writeln(C);
 // Check getNth() method
 //
 for i in 1..8 {
-  A[i] = rs4.getNth(i);
-  B[i] = rs5.getNth(i);
```

Comment:
nit: I'd probably change the loop instead of adjusting for the `-1`

Reply:
Yeah, but then I have to change all the array indexing to be i+1, in this loop, or else change all arrays and ranges in this test to subtract one from them.  This felt like a smaller diff given the goal of getting this done quickly.